### PR TITLE
fix: removed refs to log4j and commons-logging

### PR DESCRIPTION
**Pull request by loulou2u**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -416,6 +420,12 @@
                 <groupId>org.springframework</groupId>
                 <artifactId>spring-aop</artifactId>
                 <version>${spring.version}</version>
+		<exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>commons-logging</artifactId>
+                    </exclusion>
+                </exclusions>
```
Comment (ChristianMurphy) [inline pom.xml:428]:
```suggestion
                <version>${spring.version}</version>
                <exclusions>
                    <exclusion>
                        <groupId>commons-logging</groupId>
                        <artifactId>commons-logging</artifactId>
                    </exclusion>
                </exclusions>
```

**Review by ChristianMurphy — APPROVED** [review]
Thanks @loulou2u!

File: pom.xml [hunk 2]
```diff
@@ -502,7 +513,8 @@
                 <version>1.2.17</version>
                 <scope>provided</scope>
             </dependency>
-            <dependency>
+            -->
```
Comment (ChristianMurphy) [inline pom.xml:516]: is commented out section of code still needed?

File: pom.xml [hunk 3]
```diff
@@ -554,6 +566,12 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
+	     <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
```
Comment (ChristianMurphy) [inline pom.xml:572]:
```suggestion
            <artifactId>httpclient</artifactId>
            <exclusions>
                <exclusion>
                    <groupId>commons-logging</groupId>
                    <artifactId>commons-logging</artifactId>
```

**Review by ChristianMurphy — APPROVED** [review]
LGTM, thanks  @loulou2u!

**Review by bjagg — APPROVED** [review]
👍 Thanks for the clean up and PR!!

**Pull request merged**